### DISCUSSION TITLE
Ambient mode not work

### DISCUSSION THREAD

**Discussion by datlmhg** [ReVanced/revanced-discussions#923] [Question] [answered]
I turn on "Ambient mode" but it not work
![Screenshot_20230304-222334_ReVanced Extended](https://user-images.githubusercontent.com/86174857/222914899-19b65489-90d9-4cb0-b3d6-8727491240fa.jpg)

**Comment by oSumAtrIX** [comment]
Exclude patches until the error is gone, report back which one caused it

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
Ambient mode is disabled by YT if you have battery saver turned on.
